dftimewolf/lib/collectors/grr_hosts.py

```diff
@@ -176,7 +204,38 @@ def _AwaitFlow(self, client, flow_id):
       if status.state == flows_pb2.FlowContext.TERMINATED:
         self.logger.info('{0:s}: Complete'.format(flow_id))
         break
+
       time.sleep(self._CHECK_FLOW_INTERVAL_SEC)
+      if not self.skip_offline_clients:
+        continue
+
+      client_last_seen = datetime.datetime.utcfromtimestamp(
+          client.data.last_seen_at / 1000000)
+      now = datetime.datetime.utcnow()
```

Comment:
see: https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow 

```
Because naive datetime objects are treated by many datetime
 methods as local times, it is preferred to use aware datetimes
 to represent times in UTC. As such, the recommended way to
 create an object representing the current time in UTC is by
 calling datetime.now(timezone.utc).
```

and: https://docs.python.org/3/library/datetime.html#datetime.datetime.now

```
This function is preferred over today() and utcnow().

```

Reply:
Fixed to use `fromtimestamp()` and `now()` with `datetime.utc` TZ (although since we're only manipulating the dates here it doesn't really matter that much I think)